main_test.go

```diff
@@ -134,8 +134,41 @@ func TestStart(t *testing.T) {
 		exitStatus <- status
 	})
 
+	t.Run("it sends SIGTERM with signalForwardingDelay", func(t *testing.T) {
+		signalForwardingDelay := uint(2)
+		cp, err := start([]string{"sleep", "10"}, []string{"linked"}, timeout, signalForwardingDelay)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		exitStatus := make(chan int)
+		go func() {
```

Comment:
I think it is better to check the process waits for at least `signalForwardingDelay` seconds like below:

```go
go func() {
	defer close(exitStatus)

	process, _ := os.FindProcess(os.Getpid())
	process.Signal(syscall.SIGTERM)
	notLinkedContainer.KnownStatus = "STOPPED"

	want := 128 + int(syscall.SIGTERM)
	select {
	case <-exitStatus:
		t.Errorf("not wait for signalForwardingDelay")
	case <-time.After(time.Duration(signalForwardingDelay+0.5) * time.Second):
	}

	got := <-exitStatus
	if got != want {
		t.Errorf("exitStatus: got: %v, want: %v", got, want)
	}
}()
```

Reply:
That's right. I fixed https://github.com/abicky/ecswrap/pull/1/commits/defe4d8f0a9621032c348e51530a9df5d0faf20e